examples/hono-client-examples/src/main/resources/logback.xml

```diff
@@ -28,8 +28,9 @@
     <appender-ref ref="STDOUT" />
   </root>
 
+  <logger name="org.apache.kafka" level="WARN"/>
   <logger name="org.eclipse.hono.vertx.example" level="DEBUG"/>
-  <logger name="org.eclipse.hono.client" level="INFO"/>
+  <logger name="org.eclipse.hono.application.client" level="INFO"/>
```

Comment:
We should leave the entry for _org.eclipse.hono.client_ as classes such as `org.eclipse.hono.client.HonoConnection` are still being involved.